src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -727,7 +720,7 @@ namespace OpenMS
         // Load all data into memory if e.g. other file type than mzML
         if (!parsing_success)
         {
-          fh.loadExperiment(abs_filename, *peak_map_sptr, file_type, ProgressLogger::GUI);
+          fh.loadExperiment(abs_filename, *peak_map_sptr, {file_type}, ProgressLogger::GUI, true, true);
```

Comment:
hmm I vaguely recall some of your calls had 

false,false
and some true,true
passed after ProgressLogger (probably related to if logging is done)

Maybe worth a quick double check...